docs/source/adapters/site.rst

```diff
@@ -199,11 +199,11 @@ Available adapter configuration options
     | Option         | Short Description                                                                 | Requirement     |
     +================+===================================================================================+=================+
     | max_age        | The result of the `condor_status` call is cached for `max_age` in minutes.        |  **Required**   |
-    +================+===================================================================================+=================+
+    +----------------+-----------------------------------------------------------------------------------+-----------------+
```

Comment:
Corrected multiple head/body row separators in the HTCondor site adapter documentation. The table was missing otherwise as it caused an error when building the docs.